src/main/java/past/cwdt/bookstore/mapper/BookMapper.java

```diff
@@ -0,0 +1,20 @@
+package past.cwdt.bookstore.mapper;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.NullValueCheckStrategy;
+import past.cwdt.bookstore.dto.BookDto;
+import past.cwdt.bookstore.dto.CreateBookRequestDto;
+import past.cwdt.bookstore.model.Book;
+
+@Mapper(
+        componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        implementationPackage = "<PACKAGE_NAME>.impl"
+)
+public interface BookMapper {
+    BookDto toDto(Book book);
+
+    Book toBook(CreateBookRequestDto requestDto);
```

Comment:
I would suggest to use @MappingTarget for updating book. Read about this.
@BeanMapping(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
    void updateBook(CreateBookRequestDto requestDto, @MappingTarget Book book);